apache2/waf_logging/waf_log_util_internal.h

```diff
@@ -1,9 +1,62 @@
 #ifndef _WAF_LOG_UTIL_INTERNAL_HEADER
 #define _WAF_LOG_UTIL_INTERNAL_HEADER
 
-#include <google/protobuf/util/json_util.h>
-#include"waf_format.pb.h"
 #include <fstream>
```

Comment:
Seems this include is not used anywhere in this file - let's remove it.

Reply:
already removed

Reply:
Nope, it is still there in the resulting file.